app/src/main/event/views/event.html

```diff
@@ -3,32 +3,10 @@ <h3 class="title">{{event.details.name}}</h3>
 <div class="row">
     <!-- LEFT COLUMN -->
     <div class="col-md-9">
-        <button ng-click="event.openCreateGiveawayModal()" class="btn btn-primary btn-sm pull-right" style="margin: 20px 14px 0 0;"><span class="glyphicon glyphicon-plus"></span></button>
-        <h4 class="subtitle">Giveaways <span class="label label-warning">{{event.countGiveaways()}}</span></h4>
-
-        <div class="giveaways">
-            <div class="row giveaway" ng-repeat="giveaway in event.giveaways">
-                <div class="col-sm-1 text-center">
-                    <img ng-src="{{giveaway.prize.imageUrl}}" class="img-responsive" />
-                </div>
-                <div class="col-sm-5">
-                    <b>{{giveaway.amount}} &times; {{giveaway.prize.name}}</b>
-                    <br /><span class="label label-warning" ng-if="giveaway.emailRequired">Requires e-mail</span>
-                </div>
-                <div class="col-sm-4">
-                    <span ng-repeat="winner in event.winners[giveaway.id]">{{winner}}, </span>
-                </div>
-                <div class="col-sm-2 text-right" ng-if="event.canUpdateGiveaway(giveaway.id)">
-                    <button class="btn btn-xs" ng-click="event.selectGiveawayToUpdate(giveaway.id)"><span class="fui-new"></span></button>
-                    <button class="btn btn-xs" ng-click="event.deleteGiveaway(giveaway.id)"><span class="fui-trash"></span></button>
-                </div>
-            </div>
-        </div>
-
 
         <div class="draw-area" ng-show="event.canDrawGiveaway()">
             <p class="text-center">Currently drawing</p>
-            <h3 class="text-center">{{event.nextGiveaway().prize.name}}</h3>
+            <h3 class="text-center">{{event.nextGiveaway().prize.name}}</h3><br/>
```

Comment:
Minor: this space could be achieved with styles. Perhaps increase margin below? :)